process/coordinator/process.go

```diff
@@ -1476,10 +1542,22 @@ func (tc *transactionCoordinator) verifyFees(
 		totalMaxDeveloperFees.Add(totalMaxDeveloperFees, scheduledGasAndFees.DeveloperFees)
 	}
 
-	for _, miniBlock := range body.MiniBlocks {
+	if len(body.MiniBlocks) != len(header.GetMiniBlockHeaderHandlers()) {
```

Comment:
I this would be happened in the past, the code would be panicked below. Actually this is an extra protection for some malicious things which could appear in the future.

Reply:
ok, it didn't happen in the past...